script/bootstrap

```diff
@@ -1,16 +1,16 @@
-#!/bin/sh
+#!/bin/bash
 
 # script/bootstrap: Resolve all dependencies that the application requires to
 #                   run.
 
 set -e
 
-cd "$(dirname "$0")/.."
+cd "${0%/*}/.."
 
 if [ -f "Brewfile" ] && [ "$(uname -s)" = "Darwin" ]; then
   brew update
 
-  brew bundle check 2>&1 >/dev/null || {
+  brew bundle check >/dev/null 2>&1 || {
```

Comment:
`2>&1 >/dev/null` is the wrong and useless use. You still get the error message (if any) in terminal. Try:
```
ls file_does_not_exist 2>&1 >/dev/null
```